applications/tari_base_node/setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Script to download, configure and run base node
+#
+
+DOWNLOAD_PATH=https://www.tari.com/binaries/tari_0.0.9.tar.gz
+INSTALL_ROOT="${HOME}/.tari/"
```

Comment:
The executable should be on the path somewhere, like `usr/local/bin`. `.tari` is the default location for the data files.

Reply:
Will update it.

Also realized I should get the latest tari_*.tar.gz for the download path.